docs/_includes/layouts/pages/basic.njk

```diff
@@ -24,9 +24,7 @@ layout: layouts/base.njk
 {% include 'partials/component/masthead.njk' %}
 {% include 'partials/component/sidenav.njk' %}
 
-<rh-surface id="main"
-            role="main"
-            color-palette="lightest">
+<rh-surface id="main" role="main">
```

Comment:
We can remove the `<rh-surface>` here and from the layouts and just use `<main>`.  We aren't hard setting a color-palette in the surface so the user preference / setting the scheme switcher is enough.

Reply:
81b1e9f0